docs/htdocs/info/docs/api/variation/variation_tutorial.html

```diff
@@ -1264,11 +1321,10 @@ <h3>Requirement</h3>
 <li>the estimated r<sup>2</sup> value is lower than the cut-off threshold in the calculation software (the threshold is 0.05)</li>
 </ul>
 <p> Looking at other associated data will help to narrow down the reason. If there are variants in the region spanned by the LDFeatureContainer 
-with genotypes with a minor allele frequency greater than zero, then assume the r<sup>2</sup> is &lt; 0.05, which essentially means no correlation. </p>
+with genotypes with a minor allele frequency greater than zero, then assume the r<sup>2</sup> is < 0.05, which essentially means no correlation. </p>
 
 
-<br />
-<h2 id="strains">Specific strain information</h2>
+<h3 id="strains">Specific strain information</h3>
```

Comment:
I'd loose the 'With the apparition of the new technologies, one of the new functionalities that the'